.fixtures.yml

```diff
@@ -8,7 +8,7 @@ fixtures:
       ref: '1.4.2'
     stdlib:
       repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
-      ref: '4.9.0'
+      ref: '4.14.0'
```

Comment:
weird… despite this bump here,
travis is checking out this exact tag: https://github.com/puppetlabs/puppetlabs-stdlib/commit/a1c380f403d110b6c06669cc7cd49f3a3c2a258d
and yet all the errors https://travis-ci.org/voxpupuli/puppet-letsencrypt/jobs/188081421#L951 suggest that puppet3 is unable to find the function.